src/foobar/admin.py

```diff
@@ -193,22 +193,27 @@ def changelist_view(self, request, extra_context=None):
 
 
 @admin.register(models.WalletLogEntry)
-class WalletLogEntryAdmin(admin.ModelAdmin):
+class WalletLogEntryAdmin(ReadOnlyMixin, admin.ModelAdmin):
     list_display = ('date_created',
                     'user',
                     'trx_type',
-                    'post_balance',
+                    'comment',
+                    'pre_balance',
                     'amount',
-                    'pre_balance'
+                    'post_balance'
                     )
+    readonly_fields = list_display
```

Comment:
Maybe we should keep the `comment` field writeable? To avoid cluttering the timeline by forcing any spelling mistakes or similar to updated by creating a new log entry.